### ISSUE TITLE
  multiple packages link to native library `freetype`, but a native library can be linked only once

### ISSUE THREAD

**Issue by stokhos** [jwilm/alacritty#2395]
Which operating system does the issue occur on?
Linux

If on linux, are you using X11 or Wayland?
Wayland

Trying to install alacritty on clearlinux, but I got error saying:
    Updating git repository `https://github.com/jwilm/alacritty`
  Installing alacritty v0.3.2 (https://github.com/jwilm/alacritty#bc038f82)
    Updating crates.io index
    Updating git repository `https://github.com/chrisduerr/rust-clipboard`
error: failed to compile `alacritty v0.3.2 (https://github.com/jwilm/alacritty#bc038f82)`, intermediate artifacts can be found at `/tmp/cargo-installTk54PG`

Caused by:
  multiple packages link to native library `freetype`, but a native library can be linked only once

package `freetype-sys v0.7.1`
    ... which is depended on by `freetype-rs v0.19.1`
    ... which is depended on by `font v0.1.0 (https://github.com/jwilm/alacritty#bc038f82)`
    ... which is depended on by `alacritty_terminal v0.3.2 (https://github.com/jwilm/alacritty#bc038f82)`
    ... which is depended on by `alacritty v0.3.2 (https://github.com/jwilm/alacritty#bc038f82)`
links to native library `freetype`

package `servo-freetype-sys v4.0.3`
    ... which is depended on by `servo-fontconfig-sys v4.0.7`
    ... which is depended on by `servo-fontconfig v0.4.0`
    ... which is depended on by `font v0.1.0 (https://github.com/jwilm/alacritty#bc038f82)`
    ... which is depended on by `alacritty_terminal v0.3.2 (https://github.com/jwilm/alacritty#bc038f82)`
    ... which is depended on by `alacritty v0.3.2 (https://github.com/jwilm/alacritty#bc038f82)`
also links to native library `freetype`

**Comment by chrisduerr** [comment]
Duplicate of https://github.com/jwilm/alacritty/issues/2378.
